outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "username" {
+  value = { for k, v in github_membership.membership : k => v.username }
+}
+
+output "role" {
+  value = { for k, v in github_membership.membership : k => v.role }
+}
```

Comment:
why not just export `github_membership.membership` as-is?

Reply:
yeah good idea!